src/ensembl/src/shared/components/radio/Radio.scss

```diff
@@ -1,7 +1,42 @@
 @import 'src/styles/common';
 
+.radioWrapper {
+  display: block;
+  padding: 4px;
+}
+
 .radioInput {
-  display: inline-block;
-  margin-left: 20px;
-  margin-right: 20px;
+  width: 0;
+  display: none;
+}
+
+.defaultRadio {
+  height: 15px;
+  width: 15px;
+  border-radius: 50%;
+  border: 1px solid $grey;
+  background-color: none;
+  position: absolute;
+  margin-top: 4px;
+}
+
+.checked {
+  background-color: $white;
+}
+
+.checked:after {
+  top: 2px;
+  left: 2px;
+  width: 9px;
+  height: 9px;
+  border-radius: 50%;
+  background: $green;
+  display: block;
+  content: '';
+  position: absolute;
+}
+
+.radioLabel {
+  margin-left: 30px;
```

Comment:
Question to discuss with Andrea: should the distance between a checkbox and a label:

![image](https://user-images.githubusercontent.com/6834224/79223122-a4778080-7e50-11ea-8cc2-358117b03853.png)

be the same as the distance between a radio button and a label:

![image](https://user-images.githubusercontent.com/6834224/79223179-c244e580-7e50-11ea-8f89-f49318ececc4.png)

In Storybook, it seems that they are different

Reply:
Yep thats something i was thinking of confirming with Andrea once we add it to 2020 (filter and sort by component in EV). For now just setting some margin and i will confirm later if thats ok.